### PR TITLE
Standardizes readme language

### PR DESCRIPTION
**Pull request by emileighoutlaw**
[ci skip]

@leahbannon @LindsayYoung @jenniferthibault @noahmanger @jmcarp

### CODE REVIEW

**Comment by emileighoutlaw** [comment]
I have no idea what the merge conflict is here...

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,48 +1,49 @@
-## Team Processes
 
-* Use PEP8 as the coding standard for Python (but don't worry about fitting lines within 79 characters).
-* Pull requests for all commits, even typos.
-* Don't merge your own pull request. Find a friend to review your code and merge your pull request.
-* Pull requests contain some tests. Ideally they would contain decent test coverage.
-
-### Issues
+## Welcome
```
Comment (noahmanger) [inline CONTRIBUTING.md]: We should keep the code badges at the top, right?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,48 +1,49 @@
...
-* What the goal of the task being accomplished is; and
-* The user need being addressed.
+If you’d prefer, you can also reach us by [email](mailto:betafeedback@fec.gov).
 
-#### Development issues should include:
-* Unknowns tasks or dependencies that need investigation; and
-* [The boilerplate tasks for each development issue](https://gist.github.com/theresaanna/86be7e29214a7f31ab73).
-
-Use checklists (via `- [ ]`) to keep track of sub-items wherever possible.
+## Public domain
```
Comment (noahmanger) [inline CONTRIBUTING.md]: Move this to the bototm?

File: CONTRIBUTING.md (hunk 1 above)
Comment (noahmanger) [inline CONTRIBUTING.md]: Disregard. I thought I was reading README

File: README.md [hunk 3]
```diff
@@ -1,4 +1,31 @@
...
+- If you’re a developer, follow the installation instructions in the README.md page of each repository to run the apps on your computer. 
+
+## Copyright and licensing
+This project is in the public domain within the United States, and we waive worldwide copyright and related rights through [CC0 universal public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/). Read more on our license page.
+
+A few restrictions limit the way you can use FEC data. For example, you can’t use contributor lists for commercial purposes or to solicit donations. Learn more on FEC.gov.
+
+---
 
 **Develop**
```
Comment (noahmanger) [inline README.md]: This should be at the top.

File: README.md [hunk 4]
```diff
@@ -1,4 +1,31 @@
...
+- [fec-style](https://github.com/18F/fec-style): shared styles and user interface components
+- [fec-cms](https://github.com/18F/fec-cms): the content management system (CMS) for betaFEC
+
+## How you can help
+We’re thrilled you want to get involved! Here are some suggestions:
+- Check out our [contributing guidelines](https://github.com/18F/openfec/blob/master/CONTRIBUTING.md). Then, [file an issue](https://github.com/18F/fec/issues) or submit a pull request.
+- [Send us an email](mailto:betafeedback@fec.gov).
+- If you’re a developer, follow the installation instructions in the README.md page of each repository to run the apps on your computer. 
+
+## Copyright and licensing
```
Comment (noahmanger) [inline README.md]: Let's move this to the bottom?

File: README.md (hunk 3 above)
Comment (emileighoutlaw) [inline README.md]: Gar! My brain is dead, and I am struggling to understand. I'm going to ask you to talk me through this in the morning before the meeting. :)

File: README.md [hunk 5]
```diff
@@ -1,4 +1,31 @@
-# openFEC
+## Campaign finance for everyone
```
Comment (LindsayYoung) [inline README.md]: Can we add something at the top of the documentation that says that this is the repo for the API? So you can know what you are looking at without reading the full project description.

**Comment by jmcarp** [comment]
What's the status of this patch @emileighoutlaw @noahmanger @LindsayYoung? I'm happy to help resolve merge conflicts, but it looks like there are a few questions outstanding on the text as well.

**Comment by emileighoutlaw** [comment]
Working on it right now! :)

**Comment by emileighoutlaw** [comment]
Added the line per Lindsay's request. I double checked the rest of the README— all the subsections are in the same order as the other FEC repos. So I think we're ready for a merge @jmcarp

**Comment by jmcarp** [comment]
For repos that include code, it's useful (and conventional) to include the badges at the top of the README--I want to be able to open the repo and quickly understand whether tests are passing. Mind if I move the badges up while I resolve merge conflicts @emileighoutlaw?

**Comment by emileighoutlaw** [comment]
Definitely move them up. I'll open PRs for the other repos too.

Thanks for letting me know.

**Comment by jmcarp** [comment]
This has been lingering for a while and accumulating merge conflicts, so I decided to merge before all comments were resolved. @LindsayYoung and @noahmanger, could you submit patches or file issues for the issues you brought up? I definitely agree that each repo should include a brief heading at the top explaining what it is (API, data app, shared styles, etc.).

**Pull request merged**